classes/person.rb

```diff
@@ -1,14 +1,15 @@
 # ./classes/person.rb
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     super()
     @age = age
     @parent_permission = parent_permission
     @name = name
+    @rentals = []
   end
 
```

Comment:
- [ ] Please create a `rental` method and remember `the person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
done